lib/node_modules/@stdlib/stats/base/dists/logistic/ctor/benchmark/benchmark.js

```diff
@@ -20,525 +20,638 @@
 
 // MODULES //
 
-var bench = require( '@stdlib/bench' );
-var randu = require( '@stdlib/random/base/randu' );
-var isnan = require( '@stdlib/math/base/assert/is-nan' );
-var EPS = require( '@stdlib/constants/float64/eps' );
-var pkg = require( './../package.json' ).name;
-var Logistic = require( './../lib' );
+var bench = require('@stdlib/bench');
+var Float64Array = require('@stdlib/array/float64');
+var uniform = require('@stdlib/random/base/uniform');
+var isnan = require('@stdlib/math/base/assert/is-nan');
+var EPS = require('@stdlib/constants/float64/eps');
+var pkg = require('./../package.json').name;
+var Logistic = require('./../lib');
 
 
 // MAIN //
 
-bench( pkg+'::instantiation', function benchmark( bm ) {
-	var dist;
-	var mu;
-	var s;
-	var i;
+bench( pkg+'::instantiation', function benchmark( bm) {
+    var dist;
+    var len;
+    var mu;
+    var s;
+    var i;
+
+	len = 100;
+	mu = new Float64Array(len);
+	s = new Float64Array(len);
+	for (i = 0; i < len; i++) {
+		mu[i] = uniform(EPS, 10.0);
+		s[i] = uniform(EPS, 10.0);
+	}
```

Comment:
```suggestion
	mu = new Float64Array( len );
	s = new Float64Array( len );
	for ( i = 0; i < len; i++ ) {
		mu[i] = uniform( EPS, 10.0 );
		s[ i ] = uniform( EPS, 10.0 );
	}
```

Same comment as above and applies throughout the PR.